lib/common/src/Dom/Document.php

```diff
@@ -163,7 +163,7 @@ final class Document extends DOMDocument
      *
      * @var string
      */
-    const AMP_EMOJI_ATTRIBUTE_PATTERN = '/(<html [^>]*?)('
+    const AMP_EMOJI_ATTRIBUTE_PATTERN = '/(<html\s[^>]*?)('
```

Comment:
Would it be better to match multiple whitespace characters?

```suggestion
    const AMP_EMOJI_ATTRIBUTE_PATTERN = '/(<html\s+[^>]*?)('
```

Reply:
Good question, but that will already be the case with `[^>]*?`.

Reply:
Ah right, missed that. Change looks good to me, but since @schlessera worked on this before he's the right person to review this.